src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.cs

```diff
@@ -140,6 +140,7 @@ internal static void ValidateNumber(ReadOnlySpan<byte> utf8FormattedNumber)
             // because it doesn't need to deal with "NeedsMoreData", or remembering the format.
             if (utf8FormattedNumber.IsEmpty)
             {
+                //ThrowHelper.ThrowArgumentException(nameof(utf8FormattedNumber), utf8FormattedNumber);
```

Comment:
Please don't add commented out code.  If there's a ThrowHelper for this already, use it. If not, just delete the comment and be a simple throw.